src/main/webapp/dataUseLetter/DataUseLetterIndex.js

```diff
@@ -16,7 +16,8 @@ class DataUseLetterIndex extends Component {
     this.state = {
       isLoading: true,
       error: '',
-      dul: {}
+      uuid: '',
+      dul: {uuid: null}
```

Comment:
TOL: Not for this PR, but one of the areas I've had difficulty with in our current coding practice is the instantiation of state objects this way. Without the parameters defined as you have here, you see undefined errors, so this makes sense. However, this is not sustainable in the long run because every component that needs to work with a `dul` now needs to repeat this same boilerplate over and over again.

What we should be doing here instead is referring to a standardized JS model object elsewhere in the app that defines what a `dul` really is. That way any component get a state dul by calling the function. 

```
function dul() {
    return { uuid: null }
}
```